kernel/src/mm/mod.rs

```diff
@@ -130,6 +130,23 @@ fn map_kernel_rwx(mm: &mut crate::MemoryImpl, pmm: &mut PhysicalMemoryManager, p
     }
 }
 
+fn map_console(
```

Comment:
Same as above, but more different. 
We could juste pass an arrays of drivers and go over their pages.